src/database/json_convert.py

```diff
@@ -0,0 +1,100 @@
+import os
+import requests # pip install requests
+
+# The authentication key (API Key).
+# Get your own by registering at https://app.pdf.co
+API_KEY = "ymao@olin.edu_79f0f0b7a25c01d3b49b57614386df889907"
```

Comment:
Don't have api key stored explicitly

Reply:
I will send the API keys to you! The empty files are dummy files that I haven't implemented yet but will be necessary.

Reply:
Will this API key always be the same as the one in `pdf_convert.py`? If yes, update to match my comment above.